Api / Fields: Limit constraint name when adding and index to a field (#24203)
directus/directus: .changeset/sharp-apes-impress.md

---
'@directus/schema': patch
'@directus/api': patch
---

> Fixed constraint name limits not respected cross db as well as `is_indexed` returning true when only a unique index is set for MSSQL and SQLite

Reviewer: ```suggestion
Fixed constraint name limits and `is_indexed` behavior for MSSQL/SQLite
```
